src/components/RoomView/RoomView.tsx

```diff
@@ -153,14 +178,15 @@ const RoomView = (props) => {
             let spymasterId = Object.keys(spymaster);
             if (spymasterId.includes(playerId)) {
               //get set of cards with team ids from backend and set spymaster words
-              let wordsWithTeamIds = {};
+              let wordsWithTeamIds = {} as WordsWithTeamIdsObj;
               let spyWords = await axios.get(`/api/card/get25/forRoom/${roomId}`);
               spyWords.data.forEach(
-                (card) =>
+                (card: CardObj) =>
                   (wordsWithTeamIds[card.id] = {
                     id: card.id,
                     isVisibleToAll: card.isVisibleToAll,
-                    word: card.word.word,
+                    wordString: card.word.word,
```

Comment:
TS wanted us to have the wordObj and a separate wordString bc of how we're accessing the data. (i.e., we couldnt access card.word.word without accessing card.word which is an object, and needed to be typed as such. 

this is why I had to make a wordString and access it differently than we had previously in other files. :)

can go over this in class if confusing!

Reply:
np. i think this is fine. moreover, the redux state should probably be 'cards' instead of 'words' anyway